back/public/javascripts/file.js

```diff
@@ -286,11 +270,11 @@ frostbite.controller('FileCtrl', ['$scope','$http', '$q', function ($scope, $htt
     }
 
     // Parse data and attach user data to lines of code
-    $scope.add_lines = function (userData, index, length) {
+    $scope.add_lines = function (userData, length, color) {
         console.log("into add lines")
         console.log(userData);
 
-        var userColor = $scope.rainbow(length, index);
```

Comment:
doesn't look like 'length' is being used anymore in this function? Can we get rid of it?